tests/template_backends/test_jinja2.py

```diff
@@ -51,13 +51,18 @@ def test_exception_debug_info_min_context(self):
         with self.assertRaises(TemplateSyntaxError) as e:
             self.engine.get_template('template_backends/syntax_error.html')
         debug = e.exception.template_debug
-        self.assertEqual(debug['after'], '')
-        self.assertEqual(debug['before'], '')
-        self.assertEqual(debug['during'], '{% block %}')
-        self.assertEqual(debug['bottom'], 1)
-        self.assertEqual(debug['top'], 0)
-        self.assertEqual(debug['line'], 1)
-        self.assertEqual(debug['total'], 1)
+        debug_output = (
+            ('after', ''),
+            ('before', ''),
+            ('during', '{% block %}'),
+            ('bottom', 1),
+            ('top', 0),
+            ('line', 1),
+            ('total', 1),
+        )
+        for key, output in debug_output:
+            with self.subTest(key=key, output=output):
```

Comment:
I'm not sure if this is ideal usage of subTest. On a subTest failure, is it useful to continue running the rest of the test? Hard to say but it seems inconsistent to treat these assertions differently than the others just because they're repeated.

Reply:
Revert this, and the snippets for the two chunks, but keep the `subTest` use for `add_domain`? I think that at least makes sense to keep/change.

Reply:
Yes, the refactoring for `add_domain` is certainly an improvement.